sys-block/disktype/disktype-9-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Detect the content format of a disk or disk image"
+HOMEPAGE="http://disktype.sourceforge.net/"
+SRC_URI="mirror://sourceforge/disktype/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~sparc ~x86"
+
+src_compile() {
+	emake CC="$(tc-getCC)" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}"
+}
+
+src_install() {
+	dobin disktype
+	dodoc README HISTORY TODO
```

Comment:
Since those are default, you could call instead ```einstalldocs```?

Reply:
Hmm, `HISTORY` wouldn't be installed with `einstalldocs` (unless i define `DOCS` variable of course). If you don't mind i can change it of cource :)

Reply:
Hm, you are right. I was sure, that's the case for `HISTORY` too. Let's leave it in this case.